src/test/java/core/basesyntax/service/FruitServiceImplTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.fruitoperation.Operation;
+import core.basesyntax.fruitoperation.OperationBalance;
+import core.basesyntax.fruitoperation.OperationPurchase;
+import core.basesyntax.fruitoperation.OperationReturn;
+import core.basesyntax.fruitoperation.OperationSupply;
+import core.basesyntax.fruitoperation.Operations;
+import core.basesyntax.fruitoperation.strategy.OperationStrategy;
+import core.basesyntax.fruitoperation.strategy.OperationStrategyImpl;
+import core.basesyntax.service.file.DataReader;
+import core.basesyntax.service.file.DataValidator;
+import core.basesyntax.service.file.impl.DataValidatorImpl;
+import core.basesyntax.service.file.impl.FileReader;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitServiceImplTest {
+    private static FruitService fruitService;
+    private static DataValidator validator;
+    private static DataReader reader;
```

Comment:
we are testing FruitService here - it should not be dependent on reader and validator

Reply:
done